kubectl-fdb/cmd/buggify_crash_loop.go

```diff
@@ -66,23 +70,33 @@ func newBuggifyCrashLoop(streams genericclioptions.IOStreams) *cobra.Command {
 				return err
 			}
 
-			return updateCrashLoopList(kubeClient, cluster, args, namespace, wait, clear, clean)
+			return updateCrashLoopContainerList(kubeClient, cluster, containerName, args, namespace, wait, clear, clean)
 		},
 		Example: `
 # Add process groups into crash loop state for a cluster in the current namespace
 kubectl fdb buggify crash-loop -c cluster pod-1 pod-2
 
+# Add process groups into crash loop state for a cluster in the current namespace with container name
+kubectl fdb buggify crash-loop -c cluster --container-name container-name pod-1 pod-2
+
 # Remove process groups from crash loop state from a cluster in the current namespace
 kubectl fdb buggify crash-loop --clear -c cluster pod-1 pod-2
 
+# Remove process groups from crash loop state from a cluster in the current namespace with container name
+kubectl fdb buggify crash-loop --clear -c cluster --container-name container-name pod-1 pod-2
+
 # Clean crash loop list of a cluster in the current namespace
 kubectl fdb buggify crash-loop --clean -c cluster
 
+# Clean crash loop list of a cluster in the current namespace with container name
```

Comment:
We can remove the other examples, we only want to support the new way to specify the buggify setting.